viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -333,6 +333,15 @@ export class Cognite3DViewer {
     });
   }
 
+  /**
+   * Limit the number of pixels on the screen. When the render area has more pixels than this max resolution,
+   * Reveal will automatically render to an image with at most the max resolution, scaled up to fit the render area.
+   * @param maxResolution
+   */
+  setMaximumRenderResolution(maxResolution: number): void {
+    this._revealManagerHelper.revealManager.setResolutionThreshold(maxResolution);
+  }
```

Comment:
Should the constructor-option now be marked `@deprecated` for removal in 5.0, and refer to this setter instead?

Reply:
Sure, I'll change it.

Now that I think of it.. The moving-resolution can also be set dynamically? Should we have a similar setter for that one? :thinking: